software/README.asciidoc

```diff
@@ -0,0 +1,57 @@
+= Software guide
```

Comment:
Software README and other software files should be in `software` branch.

https://ut-robotics.github.io/picr22-home/team_documentation_requirements.html
https://ut-robotics.github.io/picr22-home/github_guide.html

Reply:
Added it to the software branch and created a pull request from there. Removed the software directory from Documentation branch.